webui/react/src/pages/SignIn.tsx

```diff
@@ -132,18 +134,33 @@ const SignIn: React.FC = () => {
             orientation={Orientation.Vertical}
           />
           <DeterminedAuth canceler={canceler} />
-          {info.ssoProviders?.map((ssoProvider) => {
-            const key = ssoProvider.name.toLowerCase();
-            const logo = logoConfig[key] ? <img alt={key} src={logoConfig[key]} /> : '';
-            return (
-              <Button key={key} size="large" type="primary">
-                <a className={css.ssoButton} href={samlUrl(ssoProvider.ssoUrl, ssoQueryString)}>
-                  Sign in with {logo}{' '}
-                  {ssoProvider.name === key ? capitalize(key) : ssoProvider.name}
-                </a>
-              </Button>
-            );
-          })}
+          {info.ssoProviders && info.ssoProviders.length > 0 && (
+            <>
+              <Divider>OR</Divider>
+              <Form className={css.form} layout="vertical">
+                <p>Alternatively, sign in with SSO</p>
+                {info.ssoProviders.map((ssoProvider) => {
+                  const key = ssoProvider.name.toLowerCase();
+                  const logo = logoConfig[key] ? (
+                    <img alt={key} className={css.ssoLogo} src={logoConfig[key]} />
+                  ) : null;
+                  return (
+                    <Form.Item key={key}>
+                      <Button type="primary">
+                        <a
+                          className={css.ssoButton}
+                          href={samlUrl(ssoProvider.ssoUrl, ssoQueryString)}>
+                          <Typography.Text ellipsis>
+                            {logo} {ssoProvider.name === key ? capitalize(key) : ssoProvider.name}
```

Comment:
**chore:** can you put up a screenshot for one that shows an icon? Just to make sure that there is a space between the logo and the label? I think the weird `{'  '}` was needed previously otherwise the logo would butt up against the label.

Reply:
<img width="1222" alt="image" src="https://user-images.githubusercontent.com/109113616/219801163-a3da5e7d-c9fd-4599-99f7-a19af71a8925.png">

Reply:
a BLT sounds good right now...